lib/puppet-lint/plugins/check_whitespace/multiple_whitespace.rb

```diff
@@ -0,0 +1,27 @@
+# Public: Check the manifest tokens for multiple spaces in whitespace that is not after a newline and record a warning for each instance found.
+# #
+# https://docs.puppet.com/puppet/latest/style_guide.html#spacing-indentation-and-whitespace
+PuppetLint.new_check(:multiple_whitespace) do
```

Comment:
This is going to break this syntax:

```puppet
class test (
  Boolean                   $foo = false,
  Enum['ensure', 'present'] $bar = 'ensure',
) {
}
```

We prefer it, because we think it make the parameter/data type definition easier readable.